packages/styles/scss/patterns/blocks/FeaturedLink/_featuredlink.scss

```diff
@@ -12,13 +12,18 @@
     color: $text-01;
     background: $ui-background;
 
+    &__card svg {
+      float: right;
+    }
```

Comment:
Can we also add the color change here? It's supposed to be a white arrow, and I think it makes sense to override it as a one off in this pattern since it's the only time I'm seeing it at the moment.

Reply:
yep good catch